test/gtest/ucs/test_mpool.cc

```diff
@@ -77,6 +77,30 @@ class test_mpool : public ucs::test {
     static const size_t data_size = 152;
     static const size_t align = 128;
     static size_t leak_count;
+
+    ucs_status_t setup_mpool(ucs_mpool_t *mp, size_t elem_size,
+                             unsigned elems_per_chunk, unsigned max_elems = 0)
+    {
+        static ucs_mpool_ops_t mpool_ops = {ucs_mpool_chunk_malloc,
+                                            ucs_mpool_chunk_free, NULL, NULL,
+                                            NULL};
+        if (!max_elems) {
```

Comment:
minor: coding style `if (max_elems != 0)`, as `max_elems` is a number not a boolean

Reply:
fixed

Reply:
fixed